.github/pull_request_template.md

```diff
@@ -1,6 +1,6 @@
 <!--
 Ensure your title is short, descriptive, and in the imperative mood (Fix X, Change Y, instead of Fixed X, Changed Y).
-For a good inspiration of what to write in commit messages and PRs please review https://chris.beams.io/posts/git-commit/ and our https://jellyfin.readthedocs.io/en/latest/developer-docs/contributing/ page.
+For a good inspiration of what to write in commit messages and PRs please review https://chris.beams.io/posts/git-commit/ and our documentation.
```

Comment:
Why no link to contributing part of our docs?

Reply:
I want to rewrite all that eventually anyways, both the PR template as well as that doc page, so for now it's just a vague "see our docs".